tests/cupy_tests/linalg_tests/test_decomposition.py

```diff
@@ -127,14 +127,23 @@ def check_mode(self, array, mode, dtype, batched=False):
         a_gpu = cupy.asarray(array, dtype=dtype)
         result_gpu = cupy.linalg.qr(a_gpu, mode=mode)
 
-        if ((not batched)
-                or (numpy.lib.NumpyVersion(numpy.__version__) >= '1.22.0')):
+        if numpy.lib.NumpyVersion(numpy.__version__) >= '1.22.0a1':
```

Comment:
1.22.0rc1 < 1.22.0 in NumpyVersion so we have to use a pre-released version string to test against release candidates.